src/features/InteractWithContracts/components/Interact.tsx

```diff
@@ -335,6 +335,9 @@ function Interact(props: CombinedProps) {
                   }}
                   onChange={(address) => handleAddressOrDomainChanged(address)}
                   value={values.address}
+                  setFieldValue={setFieldValue}
+                  setFieldTouched={setFieldTouched}
+                  setFieldError={setFieldError}
```

Comment:
Can't we update the field `onChange`?

Reply:
Due to complexity of `GeneralLookupField` let's handle later.